app/utils/UtilFunc.js

```diff
@@ -2,23 +2,24 @@ export const isFunc = functionToCheck =>
   functionToCheck && {}.toString.call(functionToCheck) === '[object Function]';
 
 export const formatBalance = balance =>
-  parseFloat(balance)
-    .toFixed(4)
-    .toString();
+  parseFloat(
+    `${balance.toString().split('.')[0]}.${balance
+      .toString()
+      .split('.')[1]
```

Comment:
Did you consider edge cases when balance is 0 or whole number without decimal point because this will throw an error when balance is 0.

```
formatBalance(0);
```